prow/tide/tide_test.go

```diff
@@ -442,9 +442,18 @@ func TestSetStatuses(t *testing.T) {
 		fc := &fgc{}
 		ca := &config.Agent{}
 		ca.Set(&config.Config{})
-		c := &Controller{ghc: fc, ca: ca}
-		if err := c.setStatuses([]PullRequest{pr}, pool); err != nil {
-			t.Errorf("For case %s: error setting status: %v", tc.name, err)
+		log := logrus.WithField("component", "tide")
```

Comment:
`// setStatuses logs instead of returning errors. Construct a logger to watch for errors to be printed.`